registration-processor/core-processor/registration-processor-abis-handler-stage/src/main/java/io/mosip/registration/processor/abis/handler/stage/AbisHandlerStage.java

```diff
@@ -599,6 +608,32 @@ public String getFilename() {
 
 		return response.getDataShare().getUrl();
 	}
+	private void validateBiometricRecord(BiometricRecord biometricRecord, List<String> modalities) throws DataShareException {
+		if(biometricRecord == null  || biometricRecord.getSegments() == null
+				|| biometricRecord.getSegments().isEmpty() || modalities==null || modalities.isEmpty()) {
+			throw new DataShareException("No Biometrics Found with Data Share Policy");

```

Comment:
biometricRecord null/empty and modalities null/empty is 2 different checks and needs to be 2 different errors

Reply:
Sure i'll incorporate that